components/BaseLinePlotComponent.vue

```diff
@@ -234,16 +238,46 @@ export default class BaseLinePlotComponent extends Vue {
     return this.cumulatedSplitTimes[virtualCurrentSplitIndex] || [];
   }
 
+  get goldDate() {
+    const attempts = this.parsedSplits.Run.AttemptHistory.Attempt;
+    const dateTime = attempts.find(a => this.timesToPlot[this.gold.x]['@_id'] === a['@_id'])?.['@_ended'].split(" ");
+    if (dateTime) {
+      const date = dateTime[0].split("/");
```

Comment:
You have the same 2 lines in plot_data just below, probably make a function ? (similar to these https://github.com/Webcretaire/LiveSplitAnalyzer/blob/master/util/durations.ts )